clappr/clappr-core
# Update dependencies for Node v18.17.1 LTS and Rollup v3 adjustments

## Summary

This PR updates the dependencies to align with the latest LTS version of Node, which is v18.17.1. Consequently, it also updates Rollup to version 3. To ensure existing code remains functional, it's now necessary to use the --bundleConfigAsCjs flag in calls.

## Changes

- `package.json`:
  - Add --bundleConfigAsCjs to maintain the existent code
  - version updates:
    - @babel/core updated to `^7.22.17`
    - @babel/preset-env updated to `^7.22.15`
    - @clappr/core updated to `^0.6.1`
    - @rollup/plugin-babel updated to `^6.0.3`
    - @rollup/plugin-commonjs updated to `^25.0.4`
    - @rollup/plugin-node-resolve updated to `^15.2.1`
    - @rollup/plugin-replace updated to `^5.0.2`
    - babel-jest updated to `^29.7.0`
    - coveralls updated to `^3.1.1`
    - eslint updated to `^8.49.0`
    - hls.js updated to `^1.4.12`
    - jest updated to `^29.7.0`
    - rollup updated to `^3.29.1`
    - rollup-plugin-filesize updated to `^10.0.0`
    - rollup-plugin-livereload updated to `^2.0.5`
    - rollup-plugin-serve updated to `^2.0.2`
    - rollup-plugin-sizes updated to `^1.0.5`
    - rollup-plugin-visualizer updated to `^5.9.2`
- `rollup.config.js`: 
   - Updated visualizer to use commonJS.
- remove htmlLoader
- update .test.js files

## How to test
- Switch to Node version v18.17.1 and verify that everything functions correctly.
